packages/ui-extensions/src/surfaces/admin/components/Tooltip.d.ts

```diff
@@ -0,0 +1,130 @@
+/** VERSION: 1.8.0 **/
```

Comment:
@sophschneider would you double-check tooltip types?

Reply:
you might need to `gt sync` `admin-ui-foundations`, it looks like `[overlayHidden]: boolean;` isn't private still 🤔

Reply:
Looks like it's [flagged as private where it needs to be](https://github.com/Shopify/ui-extensions/pull/3139/files#diff-4c0c209a565d1ae7b2ba318e6535d085dd682d66752404c6b332016f04e085afR94-R95). We can run the docs admin script and run dev locally to check in the morning